prover/circuits/srs_store.go

```diff
@@ -102,18 +106,52 @@ func (store *SRSStore) GetSRS(ctx context.Context, ccs constraint.ConstraintSyst
 	sizeCanonical, sizeLagrange := plonk.SRSSize(ccs)
 	curveID := fieldToCurve(ccs.Field())
 
+	var lagrangeSRS kzg.SRS
+	loadLagrangeErr := make(chan error, 2)
+	go func() { // attempt to find lagrange SRS
```

Comment:
do we really win much by loading iin parallel? Makes sense to do it, but just curious, last I checked the readDump was pretty fast

Reply:
Just a few seconds, mostly a win when debugging.